app-backend/batch/src/main/scala/stacExport/LayerCollectionBuilder.scala

```diff
@@ -0,0 +1,322 @@
+
+    // Build a scene collection and a list of scene items
+    val sceneCollectionId = UUID.randomUUID().toString()
+    val sceneCollectionBuilder = new SceneCollectionBuilder[
+      SceneCollectionBuilder.CollectionBuilder.EmptyCollection
+    ]()
+    // s3://rasterfoundry-production-data-us-east-1/stac-exports/<catalogId>/<layerCollectionId>/<sceneCollectionID>
+    val sceneCollectionAbsPath = s"${absPath}/${sceneCollectionId}"
+    // ../../../catalog.json
+    val sceneCollectionRootPath = s"../${rootPath}"
+    val sceneCollectionAbsLink = s"${sceneCollectionAbsPath}/collection.json"
+    val sceneCollectionOwnLinks = List(
+      StacLink(
+        sceneCollectionAbsLink,
+        Self,
+        Some(`application/json`),
+        Some(s"Scene Collection ${sceneCollectionId}")
+      ),
+      StacLink(
+        rootPath,
+        StacRoot,
+        Some(`application/json`),
+        Some("Root")
+      ),
+      StacLink(
+        s"${absPath}/collection.json",
+        Parent,
+        Some(`application/json`),
+        Some("Layer Collection")
+      )
+    )
+    val (sceneCollection, sceneItems, sceneItemLinks): (
+        StacCollection,
+        List[StacItem],
+        List[(String, String)]
+    ) = sceneCollectionBuilder
+      .withVersion(layerCollection.stacVersion.get)
+      .withId(sceneCollectionId)
+      .withTitle("Scene collection")
+      .withDescription(s"Scene collection in layer ${layerCollection.id.get}")
+      .withExtent(layerCollection.extent.get)
+      .withSceneList(layerCollection.sceneTaskAnnotations.get._1)
+      .withLinks(sceneCollectionOwnLinks)
+      .withParentPath(sceneCollectionAbsPath, sceneCollectionRootPath)
+      .build()
+
+    // Build a lable collection and a list of label items
+    val labelCollectionId = UUID.randomUUID().toString()
+    val labelCollectionBuilder = new LabelCollectionBuilder[
+      LabelCollectionBuilder.CollectionBuilder.EmptyCollection
+    ]()
+    // s3://rasterfoundry-production-data-us-east-1/stac-exports/<catalogId>/<layerCollectionId>/<labelCollectionId>
+    val labelCollectionAbsPath = s"${absPath}/${labelCollectionId}"
+    // ../../../catalog.json
+    val labelCollectionRootPath = s"../${rootPath}"
+    val labelCollectionAbsLink = s"${labelCollectionAbsPath}/collection.json"
+    val labelCollectionOwnLinks = List(
+      StacLink(
+        labelCollectionAbsLink,
+        Self,
+        Some(`application/json`),
+        Some(s"Label Collection ${labelCollectionId}")
+      ),
+      StacLink(
+        rootPath,
+        StacRoot,
+        Some(`application/json`),
+        Some("Root")
+      ),
+      StacLink(
+        s"${absPath}/collection.json",
+        Parent,
+        Some(`application/json`),
+        Some("Layer Collection")
+      )
+    )
+    val tasks = layerCollection.sceneTaskAnnotations.get._2
+    val itemPropsThin = layerCollection.sceneTaskAnnotations.get._4
+    val labelSpatialExtent: Extent = tasks
```

Comment:
I am now thinking about moving this part to the TaskDao and let PostGIS do the union, since, as I experimented yesterday, it was really fast in PostGIS.